os/dutils.sh

```diff
@@ -243,12 +244,21 @@ function MENU()
 function TOOLCHAIN()
 {
 	ARGS=arm-none-eabi-$2
+
+	OUTPUTFILE+="$2.txt"
+	TIMESTAMP=$(date "+%Y.%m.%d-%H.%M.%S")
+	OUTPUTFILE=$OUTPUTFILE.$TIMESTAMP
+
 	while test $# -gt 0; do
 		ARGS+=" "$3
 		shift
 	done
+
+
 	echo $ARGS
-	docker run --rm -v ${TOPDIR}:/root/tizenrt -w /root/tizenrt/os --privileged tizenrt/tizenrt:${DOCKER_VERSION} $ARGS
+	docker run --rm -v ${TOPDIR}:/root/tizenrt -w /root/tizenrt/os --privileged tizenrt/tizenrt:${DOCKER_VERSION} $ARGS | tee $OUTPUTFILE
```

Comment:
does the command ```(ex : arm-none-eabi-addr2line -e tinyara 0x0...)``` also get added to the log file? I think it would be helpful as the file name doesn't contain the binary on which the command is run.